Jenkinsfile

```diff
@@ -121,13 +121,15 @@ node {
             --create-artifacts
       """
 
-      // Run the e2e test framework
-      sh """${env.ROOT}/contrib/jenkins/run_e2e.sh \
-            --registry gcr.io/${test_project}/catalog/ \
-	    --version ${version} \
-	    --cleanup \
-	    --create-artifacts
-      """
+      ansiColor('xterm-darker-gray') {
```

Comment:
This actually fixes the colors; I installed a plugin that allowed for that support. So it should look pretty from the Console Output logs, and the Plain Text logs won't show any color codes.

It seems like it just detects the color codes, and then substitutes in a span in the HTML with appropriate styling.